provisioning/containers/novnc/nginx.conf

```diff
@@ -0,0 +1,56 @@
+user  nginx;
+worker_processes  auto;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    #gzip  on;
+
+
+      upstream websockify {
+          server 127.0.0.1:8888;
+      }
+
+    server {
+      listen       80;
+      server_name  websockify;
+
+      location /websockify {
+          proxy_pass http://websockify;
+          proxy_http_version 1.1;
+          proxy_set_header Upgrade $http_upgrade;
+          proxy_set_header Connection "Upgrade";
+          proxy_set_header Host $host;
+      }
+      
+      location / {
+          add_header Cache-Control no-cache;
+          root   /usr/share/nginx/html;
+          index  vnc.html;
+      }
+
```

Comment:
I would suggest to add a healthz location, to enable the addition of a readiness probe

```
location /healthz {
        access_log off;
        return 200 "healthy\n";
}

```